_posts/en/newsletters/2022-03-02-newsletter.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Bitcoin Optech Newsletter #189'
+permalink: /en/newsletters/2022/03/02/
+name: 2022-03-02-newsletter
+slug: 2022-03-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a new proposed `OP_EVICT` opcode and
+includes our regular sections with summaries of new releases and release
+candidates and notable changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Proposed opcode to simplify shared UTXO ownership:** developer
+  ZmnSCPxj [posted][zmnscpxj op_evict] to the Bitcoin-Dev mailing list a
+  proposal for an `OP_EVICT` opcode as an alternative to the
+  [previously proposed][news166 tluv] `OP_TAPLEAF_UPDATE_VERIFY` (TLUV)
+  opcode.  Like TLUV, `OP_EVICT` is focused on use cases where more than
+  two users share ownership of a single UTXO, such as [joinpools][topic
+  joinpools], [channel factories][topic channel factories], and certain
+  [covenants][topic covenants].  To understand how `OP_EVICT` works,
+  imagine a joinpool where a single UTXO is controlled by four users:
+  Alice, Bob, Carol, and Dan.
+
+  Today, these four users can create a P2TR (taproot) output whose
+  keypath spend allows them to use a protocol like [MuSig2][topic musig]
+  to efficiently spend that output if they all participate in creating a
+  signature.  But, if user Dan becomes unavailable or malicious, the
+  only way for Alice, Bob, and Carol to spend their funds is to have
+  prepared in advance with Dan a tree of presigned transactions---not
+  all of which need to be used, but all of which need to be ready to use
+  to ensure complete fault tolerance.
+
+  {:.center}
+  ![Illustration of combinatorial blowup when using presigned
```

Comment:
Perhaps we can link to this image by clicking on the image for slightly easier viewing of the smallish text.